spring-boot-postgresql/accelerator.yaml

```diff
@@ -0,0 +1,166 @@
+accelerator:
+  options:
+    - name: databaseName
+      inputType: text
+      defaultValue: "postgres-database"
+      label: Database Instance Name this Application will use (can be existing one in the cluster)
```

Comment:
I see you're often using `label` to express long explanations of what options do. I _think_ `description` could be more appropriate. Please test and let me know what you think.